drivers/rundeck/src/main/kotlin/com/gatehill/corebot/driver/rundeck/action/RundeckJobTriggerService.kt

```diff
@@ -37,7 +37,7 @@ class RundeckJobTriggerService @Inject constructor(private val actionDriver: Run
         try {
             call = actionDriver.buildApiClient().runJob(
                     jobId = action.jobId,
-                    executionOptions = ExecutionOptions(argString = buildArgString(args))
+                    executionOptions = ExecutionOptions(argString = buildArgString(args), asUser = trigger.username)
```

Comment:
yes @outofcoffee. asUser is passed in executionoptions. so no need to amend RundeckApi.kt